tests/ftests/048-sudo-systemd_create_scope.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# Create a systemd scope
+#
+# Copyright (c) 2022 Oracle and/or its affiliates.
+# Author: Tom Hromatka <tom.hromatka@oracle.com>
+#
+
+from cgroup import Cgroup as CgroupCli
+from cgroup import CgroupVersion
+from libcgroup import Cgroup
+from systemd import Systemd
+from run import Run
+import ftests
+import consts
+import time
+import sys
+import os
+
+SLICE = 'libcgtests.slice'
+SCOPE = '048delegated.scope'
+pid = None
+
+# Which controller isn't all that important, but it is important that we
+# have a cgroup v2 controller
+CONTROLLER = 'cpu'
+
+def prereqs(config):
+    if config.args.container:
+        result = consts.TEST_SKIPPED
+        cause = 'This test cannot be run within a container'
+        return result, cause
+
+    if CgroupVersion.get_version(CONTROLLER) != CgroupVersion.CGROUP_V2:
+        result = consts.TEST_SKIPPED
+        cause = 'This test requires cgroup v2'
+        return result, cause
+
+    result = consts.TEST_PASSED
+    cause = None
+
+    return result, cause
+
+
+def setup(config):
+    pass
+
+
+def test(config):
+    global pid
+
+    result = consts.TEST_PASSED
+    cause = None
+
+    Cgroup.create_scope(SCOPE, slice_name=SLICE)
+
+    pid = CgroupCli.get(config, cgname=os.path.join(SLICE, SCOPE), setting='cgroup.procs',
+                        print_headers=False, values_only=True)
+
+    if not Systemd.is_delegated(config, SCOPE):
+        result = consts.TEST_FAILED
+        cause = 'Cgroup is not delegated'
+
+    return result, cause
+
+
+def teardown(config, result):
+    global pid
+
+    if pid is not None:
+        Run.run(['kill', '-9', str(pid)], shell_bool=True)
+
+    if result != consts.TEST_PASSED:
+        # Something went wrong.  Let's force the removal of the cgroups just to be safe.
+        # Note that this should remove the cgroup, but it won't remove it from systemd's
+        # internal caches, so the system may not return to its 'pristine' prior-to-this-test
+        # state
+        try:
+            CgroupCli.delete(config, None, os.path.join(SLICE, SCOPE))
+        except:
```

Comment:
Python lint will complain about bare except, does `except RunError as re:` make sense?

Reply:
Yep.  Done.  Except, I did `except RunError:` since we won't be using the `re` variable